design-proposals/vhostuser-interface.md

```diff
@@ -0,0 +1,78 @@
+
+# Overview
+- User Space networking is a feature that involves packet path from the VM to the data plane running in userspace like OVS-DPDK etc, by-passing the kernel.
+- With VhostUser interface, the kubevirt VM created can utilize the dpdk virtio functionality and avoid using kernel path for the traffic when DPDK is involved, allowing usage of fast datapath and improve performance.
```

Comment:
I understand that here the main focus is dpdk, but it will be useful to see if it can apply also to other vhost user backend

Reply:
The changes made are generic, so it can also be used by VPP (vector packet processing) or any other host user backend.